site/src/Tour.jsx

```diff
@@ -35,6 +36,24 @@ const Steps = [
     placement: "auto",
     offset: 0,
   },
+  {
+    target: ".tour-layers-checkboxes",
+    content:
+      "Altering the visibility at the theme level is available, as well as at a more granular type level",
+    disableBeacon: true,
+    title: "Change Layer Visibility",
+    placement: "auto",
+    offset: 0,
+  },
+  {
+    target: ".tour-layers-pins",
+    content:
+      "This button highlights the related theme. Multiple can be selected at the same time, and the button is also available via the inspector panel.",
```

Comment:
This button sets a theme as active. Active themes are highlighted on the map and prioritized over non-active themes for click events.